src/serac/infrastructure/accelerator.cpp

```diff
@@ -24,16 +24,16 @@ std::unique_ptr<mfem::Device> device;
 void initializeDevice()
 {
   SLIC_ERROR_ROOT_IF(device, "serac::accelerator::initializeDevice cannot be called more than once");
-  device = std::make_unique<mfem::Device>();
+  // device = std::make_unique<mfem::Device>();
```

Comment:
Yes. Sorry. The current state of the PR is a little weird because of the mfem. 4.3 update.

Reply:
For now I'm putting it back in case other tests rely on this behavior, but in the future we may have to keep an eye on how mfem decides `mfem::Device` should work.